File: Move all negative elements to end/solve.cpp
void segregateElements(int arr[],int n)
    {
        std::vector<int> ans;

        // Copying positive numbers
        for (int i = 0; i < n; i++) {
            if (arr[i] > 0) {
                ans.push_back(arr[i]);
            }
        }

        // Copying negative numbers
        for (int j = 0; j < n; j++) {
            if (arr[j] < 0) {
                ans.push_back(arr[j]);
            }
        }

        // Copying vector into input array
        for (int k = 0; k < n; k++) {
            arr[k] = ans[k];
        }
    }

File: Move all negative elements to end/solve.py
class Solution:
    def segregateElements(self, arr, n):
        ans = []

        # Copying positive numbers
        for i in range(n):
            if arr[i] > 0:
                ans.append(arr[i])

        # Copying negative numbers
        for j in range(n):
            if arr[j] < 0:
                ans.append(arr[j])

        # Copying list into input array
        for k in range(n):
            arr[k] = ans[k]
